src/config/mod.rs

```diff
@@ -11,8 +11,12 @@ use crate::{manager::Registered, Error};
 #[cfg(feature = "sled-store")]
 pub mod sled;
 
+#[cfg(feature = "volatile")]
```

Comment:
I think it might be nice to have both, `volatile` can be useful to debug things, also as a starting point for others who want to create their own config store implementation.

Reply:
resolving this conversation, feel free to remove the `volatile` impl when the PR is accepted if you feel that makes sense. I don't feel strongly either way.